tools/data-model-exporter/data_model_exporter/__init__.py

```diff
@@ -0,0 +1,33 @@
+__version__ = '0.1.0'
```

Comment:
Check the spec linked in the parent ticket, I believe there is a suggested file name there. This shouldn't be named __init__.py

Reply:
Why the double `data_model_exporter` directory structure?

Reply:
when we created the directory using poetry it set it up in that way.
https://python-poetry.org/docs/basic-usage/